analyzer/src/funTest/kotlin/GitRepoTest.kt

```diff
@@ -57,7 +57,7 @@ class GitRepoTest : StringSpec() {
     }
 
     override fun afterSpec(spec: Spec) {
-        outputDir.safeDeleteRecursively(force = true)
+        outputDir.safeDeleteRecursively()
```

Comment:
Please note that on Windows the files within a `.git` directory are created as read-only files, and such they can only be deleted with `force`. So I'd expect this spec to now fail to perform the cleanup on Windows, as well as all other tests that need to delete `.git` directories.

Reply:
Thanks @schuberth. So my original goal of making the calls `consistent` cannot be achieved with `force=false` thus abandon this PR.